src/global/styles/mixins/buttons.scss

```diff
@@ -63,6 +63,10 @@
   pointer-events: var(--sbb-button-pointer-events);
   background-color: transparent;
   color: var(--sbb-button-color-default-text);
+  outline-offset: var(--sbb-focus-outline-offset);
```

Comment:
Is this line really needed?

Reply:
It kinda is to make sure the animation is okay, but I refactored the style anyway in a more correct way